### ISSUE TITLE
Product Detail Page

### ISSUE THREAD

**Issue by herzog31** [hlxsites/maidenform#11]
- [x] Depends on setup of Adobe Commerce catalog service https://experienceleague.adobe.com/docs/commerce-merchant-services/catalog-service/overview.html
- [x] Validate Catalog Service setup
    - Access via https://secretshare.corp.adobe.com/?2570037164fcc37d&k=GBdUovCzuvUd2kUpaDsk2BFZjhq3HbJStZ4hP2qCKX7P
- [x] Implement Product Detail Page block
    - [x] Use catalog service API for as much as possible, if needed fallback to monolith GraphQL
    - [x] Gallery
    - [x] Product description
    - [x] Variant selection
    - [x] Quantity
    - [x] Add to cart button

Out of scope:
* https://github.com/hlxsites/maidenform/issues/12
* https://github.com/hlxsites/maidenform/issues/13
* https://github.com/hlxsites/maidenform/issues/181

**Comment by herzog31** [comment]
Currently blocked by missing catalog service.

**Comment by sgonzalezmangana** [comment]
@herzog31 last comment shows this ticket as blocked, could you confirm this is not blocked any longer?
